packages/settingeditor/src/splitpanel.ts

```diff
@@ -1,26 +0,0 @@
-/* -----------------------------------------------------------------------------
-| Copyright (c) Jupyter Development Team.
-| Distributed under the terms of the Modified BSD License.
-|----------------------------------------------------------------------------*/
-
-import { ISignal, Signal } from '@lumino/signaling';
-import { SplitPanel as SPanel } from '@lumino/widgets';
-
-/**
- * A deprecated split panel that will be removed when the phosphor split panel
- * supports a handle moved signal. See https://github.com/phosphorjs/phosphor/issues/297.
- */
-export class SplitPanel extends SPanel {
```

Comment:
Since this was exported before, maybe we should tag this PR as `api-change` and mention it in https://jupyterlab.readthedocs.io/en/latest/extension/extension_migration.html#jupyterlab-3-x-to-4-x?

Reply:
It was not exported in the `index.ts` for `settingeditor`, it was not intended to be public API. A user would have had to import the file directly from `lib` or something ... I don't consider this a public API, though. What do you think?

The intention was always to remove this so it wasn't exposed.

Reply:
Ah indeed, I just checked in the code base and it is indeed the case.

Thanks for the context!